lifelines/fitters/mixture_cure_fitter.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import autograd.numpy as anp
+from autograd.scipy.special import expit
+from lifelines.fitters import ParametricUnivariateFitter
+
+# What should the name of the fitter actually be?
+class MixtureCureFitter(ParametricUnivariateFitter):
```

Comment:
Part of me wants to call this `"Wrapper"`, to distinguish its unique API from traditional `Fitters` - but it is still a `Fitter`. Not sure..

Reply:
I was thinking of the way sklearn ensembles work when building this. For instance, the [AdaBoostClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostClassifier.html#sklearn.ensemble.AdaBoostClassifier) takes a base_estimator and is still called a Classifier. Thoughts?

Reply:
good with me then!